pkg/filesystem/local_directory_unix.go

```diff
@@ -460,3 +462,50 @@ func (d *localDirectory) Apply(arg interface{}) error {
 		return syscall.EXDEV
 	}
 }
+
+func (d *localDirectory) Mount(mountpoint path.Component, source string, fstype string, options int) error {
+	mfd, err := mountat(d.fd, fstype, source, mountpoint.String(), options)
+	if err != nil {
+		return util.StatusWrap(err, "Mountat")
+	}
+	defer unix.Close(mfd)
+
+	return nil
+}
+
+// Fchdir changes the current working directory to the directory file descriptor.
+// If there is an error, it will be of type *PathError.
+func Fchdir(dfd int) error {
```

Comment:
This is a copy of `unix.Chdir` without the logging. As `Fchdir` does not have a wrapper function.

Reply:
Do we need this function at all? Why don't we just call `Fchdir()` directly?